### ISSUE TITLE
Documentation is missing 

### ISSUE THREAD

**Issue by ashokmca07** [wttech/bobcat#479]
https://cognifide.github.io/bobcat/ is leading to 404.

**Comment by ashokmca07** [comment]
It should be https://wttech.github.io/bobcat/

**Comment by sManWorks** [comment]
Is this project even under active development?

**Comment by Shaihuludus** [comment]
No, it was retired a few months ago https://wttech.blog/blog/2021/bobcat-retirement/
